virttest/qemu_devices/qcontainer.py

```diff
@@ -618,8 +618,8 @@ def simple_hotplug(self, device, monitor, bus=None):
                         if _bus.bus_item == 'bus':
                             bus = _bus
                             break
-                if bus:
-                    bus.prepare_hotplug(device)
+            if bus is not None:
```

Comment:
I think `else:` should take care,

Reply:
@balamuruhans else, it should execute L#623 directly, to hotplug device without 'bus' option, this means let qemu decides its bus, which is intended. Thanks.

Reply:
ah... got it. Thank you.